packages/issue-receiver/src/main.ts

```diff
@@ -0,0 +1,48 @@
+import * as core from '@actions/core'
+import * as gh from '@actions/github'
+import * as Webhooks from '@octokit/webhooks'
+
+// TODO: Generalise this to work on any object
+interface Notification {
+    'list-ops': Exercise
+}
+
+interface Exercise {
+    commit_sha: string
+    commit_message: string
+    pull_request_html_url: string
+    new_cases: string[]
+}
+
+async function run(): Promise<void> {
+    try {
+        // Confirm that it's a repository_dispatch event
+        if (gh.context.eventName != 'repository_dispatch') {
+            throw new Error(`Event ${gh.context.eventName} is not supported`)
```

Comment:
```suggestion
            throw new Error(`Event ${gh.context.eventName} is not supported. Expected "repository_dispatch".`)
```

Makes it easier to act upon receiving this error.